README.md

```diff
@@ -18,3 +18,152 @@ arduino-cli upload -b rp2040:rp2040:rpipico -v . -p /dev/ttyACM0
 arduino-cli compile -b rp2040:rp2040:rpipico -v . --build-property compiler.cpp.extra_flags="-DCYPHAL_NODE_INFO_GIT_VERSION=0x$(git rev-parse --short=16 HEAD)"
 ```
 Adding argument `--build-property compiler.cpp.extra_flags="-DCYPHAL_NODE_INFO_GIT_VERSION=0x$(git rev-parse --short=16 HEAD)"` allows to feed the Git hash of the current software version to [107-Arduino-Cyphal](https://github.com/107-systems/107-Arduino-Cyphal) stack from where it can be retrieved via i.e. [yakut](https://github.com/opencyphal/yakut).
+
+## Register list
+
+| **name**                                  | **type** | **default value**                     | **Description**                        |
```

Comment:
Btw. I know you did a ton of work writing up this register interface. But is there any value in having it on the README? You can just as well get it via `yakut` and store it as a YAML, i.e.

```bash
y rl 10 | y --yaml rb > pico-base-config.yaml
```

Reply:
ninja fix

```diff
-y rl 10 | y --yaml rb > pico-base-config.yaml
+y rl 10, | y --yaml rb > pico-base-config.yaml
```
or
```diff
-y rl 10 | y --yaml rb > pico-base-config.yaml
+y rl 10 | y --yaml rb 10 > pico-base-config.yaml
```

Reply:
> Btw. I know you did a ton of work writing up this register interface. But is there any value in having it on the README? You can just as well get it via `yakut` and store it as a YAML, i.e.
> 

This is more of a documentation of the device or firmware. So that the future user knows what he/she gets. And i haven't found a way to add some comments/additional information to the yaml file.

Reply:
Allright. But keep in mind that's now in your responsibility to keep the documentation and the code in sync. (Shouldn't be too much of a problem, because hopefully the interface is done now).

Reply:
I will surely do.
I also don't think there will be many updates as all the functions of the board are now supported. New functions or uses of this board will require new firmware based on this one.